pkg/volume/flexvolume/plugin.go

```diff
@@ -134,8 +133,6 @@ func (plugin *flexVolumePlugin) GetVolumeName(spec *volume.Spec) (string, error)
 		return "", err
 	}
 
-	klog.Warning(logPrefix(plugin), "GetVolumeName is not supported yet. Defaulting to PV or volume name: ", name)
```

Comment:
Why not reduce verbosity of this message instead of removing it altogether?

Reply:
> Why not reduce verbosity of this message instead of removing it altogether?

There is default volumeName log in GetVolumeName:

```
klog.V(4).Infof(logPrefix((*flexVolumePlugin)(plugin)), "using default GetVolumeName for volume ", spec.Name())
```

And, the message "GetVolumeName is not supported yet. Defaulting to PV or volume name: " is not suitable here, this log will be printed out when GetVolumeName is supported by plugin.

Reply:
@fredkan GetVolumeName is not supported on the K8S side, not on the plugin side. I would recommend to just adjust the verbosity of the logs, unless there is a big need to.

Reply:
Agree. Thanks.